oUF_ArtifactPower.lua

```diff
@@ -146,6 +147,10 @@ local function Enable(self, unit)
 		element:SetStatusBarColor(.901, .8, .601)
 	end
 
+	if element:IsObjectType('StatusBar') and element.colorArtifactQuality then
+		element:SetStatusBarColor(GetItemQualityColor(6))
+	end
+
```

Comment:
I'm against that as it is very limiting. Some refactoring in the lines above it solve the lack of options for coloring. I'll fix this myself, thanks for pointing out the use case.